data_handler.py

```diff
@@ -246,7 +246,8 @@ def filter_industry_codes(data: dict) -> bool:
         Checks if row 'data' has a valid industry code.
         """
         include_codes = ['561010', '561020', '563000']
-        res = 'industry_code' in data.keys() and data['industry_code'] in include_codes
+        res = 'industry_code' in data.keys(
+        ) and data['industry_code'] in include_codes
```

Comment:
I think your linter is set to 80 char line length. I'd prefer 100.

Reply:
How to change? Flake8 on vscode

Reply:
Set `max-line-length = 100` in your flake8 config. Should be in `C:\Users\<username>\.flake8`